src/cli/isready.rs

```diff
@@ -3,8 +3,10 @@ use surrealdb::engine::any::connect;
 
 #[tokio::main]
 pub async fn init(matches: &clap::ArgMatches) -> Result<(), Error> {
-	// Set the default logging level
-	crate::cli::log::init(0);
+	// Initialize o11y
+	crate::o11y::builder().with_log_level(
+		matches.get_one::<String>("log").unwrap()
+	).init();
```

Comment:
The `--log` argument isn't customisable on the `surreal isready` command. It was set to no logging.

Reply:
what's the reasoning for not allowing log customisation there? all these commands may have errors (think connection errors in the specific case) and we may want to print DEBUG messages if so. My comment apply to the rest of subcommands, I guess the original reason was the same for all of them

Reply:
For the `isready` command, the intention is to only succeed (with no errors) if the database is able to be connected to and health. Everything else is an error - so I'm not sure whether it's worth outputting any errors here. What do you think?

Reply:
Asa side note, there currently is no `--log` argument for the `surreal isready` command, so this wouldn't work anyway. If a user specifies `--log` on the `isready` command, it will return an error saying that the argument is not recognised here.

Reply:
oh, my bad, I assumed `--log` was a global arg. Will revert my changes.

In any case, we can always fine tune the logging filters by using the `SURREAL_LOG_FILTER` env var, so no need to introduce the `--log` arg to the other subcommands.

Reply:
> Everything else is an error - so I'm not sure whether it's worth outputting any errors here. What do you think?

If a user faces an error running `isready` is still useful to know exactly why. A DNS lookup error, a TLS handshake failure, a connection error or the DB not executing the query successfully, all signal different problems. If we don't print anything, the user may wrongly assume that the database is misbehaving, but it might not be true.

The original code set the log level to WARN, but ERROR might be good enough here